detox/e2e/support/server_api/common.js

```diff
@@ -2,7 +2,14 @@
 // See LICENSE.txt for license information.
 
 export const getResponseFromError = (err) => {
-    const {response: {data, status}} = err;
+    let data;
+    let status;
+
+    const {response} = err;
+    if (response) {
+        data = response.data;
+        status = response.status;
+    }
```

Comment:
Heads up that this might need to revert in favor of https://github.com/mattermost/mattermost-mobile/pull/4852/files.

Reply:
Yep! This one went out by mistake, sorry about that. Just reverted the changes!